ads/nativo.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript} from '../src/3p';
+
+export function nativo(global,data) {
+
+  let ntvAd;
+  (function() {
+    window
+      .frames
+      .location
+      .hash = window.frames.location.hash.replace(/({).*(})/,'');
+
+    ///
+    // Private
+    ///
+    let delayedAdLoad = false,
+      percentageOfadViewed = undefined;
+    const loc = window.context.location;
+
+    function isValidDelayTime(delay) {
```

Comment:
There is a lot of code here. Just in general: You may want to consider loading an AMP specific JS file instead of your generic one that has this logic, so you can change it without sending us a PR. This is up to you.